apps/storefront/src/pages/registered/RegisteredFinish.tsx

```diff
@@ -71,7 +71,7 @@ export default function RegisteredFinish(props: {
       return (
         <StyleTipContainer>
           {b3Lang('global.registerFinish.bcSuccess.tip', {
-            storeName,
+            storeName: storeName ?? '',
```

Comment:
This is not worse than passing `null` to the translator. We should probably refactor the code so `storeName` is defined when getting to this part of the code.